src/common/Common.ts

```diff
@@ -142,18 +129,16 @@ export class CommandLineUtils {
                         validate: CommandLineUtils.validateApiLevelFlag
                     })
                 };
-            case FlagsConfigType.Platform:
+            case FlagsConfigType.PlatformType:
                 return {
-                    platform: Flags.string({
+                    platform: Flags.option({
                         char: 'p',
-                        description: supportsDesktop
-                            ? messages.getMessage('platformFlagIncludingDesktopDescription')
-                            : messages.getMessage('platformFlagMobileOnlyDescription'),
-                        required: isRequired,
-                        validate: supportsDesktop
-                            ? CommandLineUtils.validatePlatformFlagIncludingDesktop
-                            : CommandLineUtils.validatePlatformFlagMobileOnly
-                    })
+                        description: messages.getMessage('platformFlagDescription'),
```

Comment:
Do you still not want to say it in here
![image](https://github.com/forcedotcom/lwc-dev-mobile-core/assets/42476999/0e86aab0-58fc-4088-82a2-48eaf04d1d58)

Something like "Specify a platform: iOS, Android, or Desktop"? Or is it made generic so that more platforms are added in the future?

Reply:
It is made generic b/c `SfCommand` now automatically adds the list of options:
```
  -p, --platform=<option>  (required) Specify platform.
                           <options: ios|android>
```